docs/UsersGuide/source/BuildingRunningTesting/WE2Etests.rst

```diff
@@ -38,15 +41,30 @@ For convenience, the WE2E tests are currently grouped into the following categor
 * ``grids_extrn_mdls_suites_nco``
    This category of tests ensures that the workflow running in **NCO mode** (i.e., with ``RUN_ENVIR`` set to ``"nco"``) completes successfully for various combinations of predefined grids, physics suites, and input data from different external models. Note that in NCO mode, an operational run environment is used. This involves a specific directory structure and variable names (see :numref:`Section %s <NCOModeParms>`).
 
+* ``ufs_case_studies``
+   This category tests that the workflow running in community mode completes successfully when running cases derived from the `ufs-case-studies repository <https://github.com/dtcenter/ufs-case-studies>`__. 
+
 * ``verification``
    This category specifically tests the various combinations of verification capabilities using METPlus. 
 
-* ``release_SRW_v1``
-   This category was reserved for the official "Graduate Student Test" case for the Version 1 SRW code release.
-
 * ``wflow_features``
    This category of tests ensures that the workflow completes successfully with particular features/capabilities activated.
 
+.. note::
+
+   Users should be aware that some tests assume :term:`HPSS` access. 
+   
+      * ``custom_ESGgrid_Great_Lakes_snow_8km`` and ``MET_verification_only_vx_time_lag`` require HPSS access, as well as ``rstprod`` access on both :term:`RDHPCS` and HPSS. 
+      * On certain machines, *community* and *nco* tests assume HPSS access. If the ``ush/machine/*.yaml`` file contains the following lines, and these paths are different from what is provided in ``TEST_EXTRN_MDL_SOURCE_BASEDIR``, users will need to have HPSS access or modify the tests to point to another data source:
```

Comment:
@mkavulich - Unfortunately, running the `community` test on Jet causes the test to fail when HPSS is down because it is looking for staged data in:
`/public/data/grids/gfs/0p25deg/grib2`
rather than:
`/mnt/lfs4/HFIP/hfv3gfs/role.epic/UFS_SRW_data/develop/input_model_data`

When the test is unable to find the data on disk, it attempts to pull the data from HPSS, which results in failure due to HPSS being down for maintenance.  The data is staged on the machines, it is simply a case that the wrong directory is being used, resulting in the failure.  Updating the `/public/data/grids/gfs/0p25deg/grib2` directory on Jet to point to the appropriate directory in `/mnt/lfs4/HFIP/hfv3gfs/role.epic/UFS_SRW_data/develop/input_model_data` should correct the issue.

Having said that, it does look like the `nco` test does successfully run without HPSS access on machines, so that test can be removed from the list.
```suggestion
      * On certain machines, the *community* test assumes HPSS access. If the ``ush/machine/*.yaml`` file contains the following lines, and these paths are different from what is provided in ``TEST_EXTRN_MDL_SOURCE_BASEDIR``, users will need to have HPSS access or modify the tests to point to another data source:
```

Reply:
Thanks for the reply @MichaelLueken. This seems like a bug to me, the expected data workflow should check in both locations, not just one and fall back to HPSS. Is there an open issue on this?

Reply:
@mkavulich - I have opened issue #925 for this issue.